src/panels/GettingStartedPanel/Steps/NavigationSteps.tsx

```diff
@@ -76,7 +76,9 @@ export const NavigationSteps: React.JSX.Element[] = [
     <Text>
       Open the navigation menu and click on the "Focus" button to focus on the Moon. The
       navigation menu can be found in the bottom bar:
-      <OriginPanelMenuButton onClick={() => {}} />
+      <Box mt={'md'} p={'md'}>
+        <OriginPanelMenuButtonContent />
+      </Box>
```

Comment:
I think it would be good with some border around these buttons, to make it clearer that it is one entity. How about:
```suggestion
      <Paper mt={'md'} p={'xs'} w={'fit-content'} withBorder bg={'transparent'}>
        <OriginPanelMenuButtonContent />
      </Paper>
```
![image](https://github.com/user-attachments/assets/34497aff-cb2d-4677-833d-e4879f035b3c)

or 
```suggestion
      <Paper mt={'md'} py={'xs'} px={'md'} w={'fit-content'}>
        <OriginPanelMenuButtonContent />
      </Paper>
```
![image](https://github.com/user-attachments/assets/f0d1574f-1b05-4201-939b-144ccb8b569b)

? 

And maybe make this wrapper a helper component that we can apply to the other buttons we use here as well? 

I already did something similar for the `SceneGraphNodeHeader` in COntentSteps.tsx, for example. That could also use the wrapper component